server/subscribe.go

```diff
@@ -92,13 +107,39 @@ type Subscriptions struct {
 	Channel       *ChannelSubscriptions
 }
 
+type ListSubscriptionTemplate map[string]*SubscriptionTemplate
```

Comment:
This is a map and not a list. Maybe name this `SubscriptionTemplateCollection`, or `SubscriptionTemplatesByID` since I think ids are what the keys are